spec/emitter_spec.rb

```diff
@@ -729,9 +728,45 @@
         "thread=c at=start\n",
         "line=3\n",
         "thread=c at=finish elapsed=0.0000\n"
-      )
+      ) # the line "thread=e line=6" is not here b/c that was in a `#silence` block
```

Comment:
Maybe adding an assert on that, to reinforce the expectations, would be useful. Something like: `expect(lines).not_to contain("thread=e line=6")`

Reply:
The `contains_exactly` matcher will assert that for us. The comment is meant for future humans who are confused by the line not showing up in the assertion. i.e., I was confused by this b/c I missed the `#silence` block the fist several times.